x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -947,7 +947,12 @@ public enum Cap {
         /**
          * Supercedes {@link Cap#MAKE_NUMBER_OF_CHANNELS_CONSISTENT_WITH_LAYOUT}.
          */
-        FIX_REPLACE_MISSING_FIELD_WITH_NULL_DUPLICATE_NAME_ID_IN_LAYOUT;
+        FIX_REPLACE_MISSING_FIELD_WITH_NULL_DUPLICATE_NAME_ID_IN_LAYOUT,
+
+        /**
+         * Fix https://github.com/elastic/elasticsearch/issues/125850
```

Comment:
I'd prefer a short description of the bug so I don't have to switch a web browser when scanning the code. Just a sentence like "fix double release bug for nulls" - I mean, that's the name. But still, I don't real the name so much as I read the comment....